src/geo/geohash.clj

```diff
@@ -83,8 +86,8 @@
     [origin]
     ; We build the list backwards by recurring from last to first,
     ; counterclockwise.
-    (let [len   (* 4 (dec n))      ; Total sequence length
-          west  0                  ; We start (at L), with i = 0, going west
+    (let [;len   (* 4 (dec n))     ; Total sequence length
```

Comment:
Let's remove these commented lines

Reply:
There's a couple of places across the PR that have things commented out. I just didn't want to be too bold in deleting unused things entirely, but I'll apply this across the board.